src/components/proxy/packet_router.rs

```diff
@@ -87,17 +85,10 @@ impl<P: PacketMut> DownstreamPacket<P> {
         );
 
         let timer = metrics::processing_time(metrics::READ).start_timer();
-        match self.process_inner(config, sessions, destinations) {
-            Ok(()) => {
-                error_acc.maybe_send();
-            }
-            Err(error) => {
-                let discriminant = error.discriminant();
-                metrics::errors_total(metrics::READ, discriminant, &metrics::EMPTY).inc();
-                metrics::packets_dropped_total(metrics::READ, discriminant, &metrics::EMPTY).inc();
-
-                error_acc.push_error(error);
-            }
+        if let Err(error) = self.process_inner(config, sessions, destinations) {
```

Comment:
I'm not sure I understand, the PR title doesn't match, this seems like errors are still being logged?

Reply:
Sorry to clarify, they're being "logged" through prometheus metrics yes, they're not being sent to stderr though. This PR is about cleaning up our tracing logging.